charts/victoria-logs-single/values.yaml

```diff
@@ -0,0 +1,265 @@
+    # -- Create/use Persistent Volume Claim for server component. Empty dir if false
+    enabled: false
+
+    # -- Array of access modes. Must match those of existing PV or dynamic provisioner. Ref: [http://kubernetes.io/docs/user-guide/persistent-volumes/](http://kubernetes.io/docs/user-guide/persistent-volumes/)
+    accessModes:
+      - ReadWriteOnce
+    # -- Persistant volume annotations
+    annotations: {}
+
+    # -- StorageClass to use for persistent volume. Requires server.persistentVolume.enabled: true. If defined, PVC created automatically
+    storageClass: ""
+
+    # -- Existing Claim name. If defined, PVC must be created manually before volume will be bound
+    existingClaim: ""
+
+    # -- Bind Persistent Volume by labels. Must match all labels of targeted PV.
+    matchLabels: {}
+
+    # -- Mount path. Server data Persistent Volume mount root path.
+    mountPath: /storage
+    # -- Mount subpath
+    subPath: ""
+    # -- Size of the volume. Should be calculated based on the logs you send and retention policy you set.
+    size: 3Gi
+
+  # -- Sts/Deploy additional labels
+  extraLabels: {}
+  # -- Pod's additional labels
+  podLabels: {}
+  # -- Pod's annotations
+  podAnnotations: {}
+  # -- Pod's management policy
+  podManagementPolicy: OrderedReady
+
+  # -- Resource object. Ref: [http://kubernetes.io/docs/user-guide/compute-resources/](http://kubernetes.io/docs/user-guide/compute-resources/
+  resources:
+    {}
+    # limits:
+    #   cpu: 500m
+    #   memory: 512Mi
+    # requests:
+    #   cpu: 500m
+    #   memory: 512Mi
+
+  # Indicates whether the Container is ready to service requests. If the readiness probe fails, the endpoints controller removes the Pod's IP address from the endpoints of all Services that match the Pod. The default state of readiness before the initial delay is Failure. If a Container does not provide a readiness probe, the default state is Success.
+  readinessProbe:
+    httpGet:
+      path: /health
+      port: http
+    initialDelaySeconds: 5
+    periodSeconds: 15
+    timeoutSeconds: 5
+    failureThreshold: 3
+
+  # Indicates whether the Container is running. If the liveness probe fails, the kubelet kills the Container, and the Container is subjected to its restart policy. If a Container does not provide a liveness probe, the default state is Success.
+  livenessProbe:
+    httpGet:
+      scheme: HTTP
+      path: /health
+      port: 9428
+    initialDelaySeconds: 30
+    periodSeconds: 30
+    timeoutSeconds: 5
+    failureThreshold: 10
+
+  # Indicates whether the Container is done with potentially costly initialization. If set it is executed first. If it fails Container is restarted. If it succeeds liveness and readiness probes takes over.
+  startupProbe: {}
+    #tcpSocket:
+    #  port: http
+    #failureThreshold: 30
+    #periodSeconds: 15
+    #successThreshold: 1
+    #timeoutSeconds: 5
+
+  # -- Security context to be added to server pods
+  securityContext: {}
```

Comment:
what do you adding the default security context and specify it as NON-ROOT?

Reply:
Added sane dafaults here.